straxen/plugins/veto_events.py

```diff
@@ -240,27 +240,26 @@ class nVETOEventPositions(strax.Plugin):
     __version__ = '0.0.1'
 
     def setup(self):
-        if os.path.exists(self.config['nveto_pmt_position_map']):
-            print(f"Path is local. Loading {self.config['nveto_pmt_position_map']} from disk. ")
-            self.path_to_map = self.config['nveto_pmt_position_map']
+        if isinstance(self.config['nveto_pmt_position_map'], str):
+            if os.path.exists(self.config['nveto_pmt_position_map']):
+                # Deprecated support, let's remove this soon
+                warnings.warn(
+                    f"Path is local. Loading "
+                    f"{self.config['nveto_pmt_position_map']} from disk.",
+                    DeprecationWarning)
+                path_to_map = self.config['nveto_pmt_position_map']
+                npmt_pos = pd.read_csv(path_to_map, index_col=0)
```

Comment:
@WenzDaniel I don't see much need for keeping this around, let's remove it?

Reply:
No I will remove it, I just followed the peaks position reconstruction as an example.